### ISSUE TITLE
How to import celltool into Python? 

### ISSUE THREAD

**Issue by huayicodes** [tdsmith/celltool#1]
I've tried adding it to the PYTHONPATH. But this somehow disabled my ipython/spyder without allowing me to load celltool.

**Comment by tdsmith** [comment]
You should run `pip install .`.

**Comment by tdsmith** [comment]
You should run `pip install .` in the directory where you downloaded it. You shouldn't put the source tarball on PYTHONPATH.

**Comment by huayicodes** [comment]
I got an error when installing with "pip install". It seems that numarray is no longer supported. How should I proceed?

pip install file:///Users/Huayi/Downloads/celltool-master.zip celltool
Processing c:\users\huayi\downloads\celltool-master.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\huayi\appdata\local\temp\pip-wb4nc9-build\setup.py", line 46, in <module>
        setup_package()
      File "c:\users\huayi\appdata\local\temp\pip-wb4nc9-build\setup.py", line 43, in setup_package
        *_setup_kws)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\core.py", line 132, in setup
        config = configuration()
      File "c:\users\huayi\appdata\local\temp\pip-wb4nc9-build\setup.py", line 24, in configuration
        config.add_subpackage('celltool')
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 1000, in add_subpackage
        caller_level = 2)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 969, in get_subpackage
        caller_level = caller_level + 1)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 906, in _get_configuration_from_setup_py
        config = setup_module.configuration(_args)
      File "celltool\setup.py", line 11, in configuration
        config.add_subpackage('numerics')
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 1000, in add_subpackage
        caller_level = 2)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 969, in get_subpackage
        caller_level = caller_level + 1)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 906, in _get_configuration_from_setup_py
        config = setup_module.configuration(*args)
      File "celltool\numerics\setup.py", line 23, in configuration
        config.add_subpackage('ndimage')
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 1000, in add_subpackage
        caller_level = 2)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 969, in get_subpackage
        caller_level = caller_level + 1)
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\distutils\misc_util.py", line 882, in _get_configuration_from_setup_py
        ('.py', 'U', 1))
      File "C:\Users\Huayi\Anaconda2\lib\site-packages\numpy\compat\py3k.py", line 143, in npy_load_module
        mod = imp.load_module(name, fo, fn, info)
      File "celltool\numerics\ndimage\setup.py", line 3, in <module>
        from numpy.numarray import get_numarray_include_dirs
    ImportError: No module named numarray

```
----------------------------------------
```

Command "python setup.py egg_info" failed with error code 1 in c:\users\huayi\appdata\local\temp\pip-wb4nc9-build\

**Comment by tdsmith** [comment]
Oh, good point! You need antediluvian versions of numpy and scipy in the virtualenv where you install celltool. Try numpy 1.7.

**Comment by huayicodes** [comment]
I have numpy 1.7 now. The error now become "  
 'gcc' is not recognized as an internal or external command,
  operable program or batch file.
  Looking for python27.dll
  Building import library (arch=AMD64): "C:\Users\Huayi\Anaconda2\libs\libpython27.a" (from C:\Users\Huayi\Anaconda2\python27.dll)
  error: [Error 2] The system cannot find the file specified". 

What I have tried and didn't work: 
1. I have installed mingw32 and added the path to mingw32\bin and mingw\gcc.exe to my PATH. 
2. calling installation with mingw as compiler: "pip install --global-option build_ext --global-option --compiler=mingw32 celltool-master.zip". 

Running out of ideas....
